### PR TITLE
Feature/add sentry team tag

### PR DESCRIPTION
**Pull request by thiagoazeredo-quintoandar**
Allow define a team tag on sentry to identify the team that owns the failure

### CODE REVIEW

File: src/transports.js [hunk 1]
```diff
@@ -54,6 +54,9 @@ class SentryTransport extends Transport {
             if (info.traceId) {
                 scope.setTag("traceId", info.traceId)
             }
+            if(info.team) {
```
Comment (imteekay) [inline src/transports.js]:
```suggestion
            if (info.team) {
```

**Review by imteekay — APPROVED** [review]
:rocket:

File: README.md [hunk 2]
```diff
@@ -103,6 +103,11 @@ At your console, the logs now contain the trace-id identifier:
 [info] Some info about processing cool object with id 11 [trace-id: TRACER-ID] { extra: { id: 11, someInfo: 'someInfo' } }, module: 'path/to/my/file.js', timestamp: '2020-06-09T22:46:21.759Z'}
 ```
 
+### Team
+We have added a new concept that allow define a team that own the error.
```
Comment (evandrocarrenho) [inline README.md:107]: Minor english fixes "... that **allows defining** a team that **owns** ... "

File: README.md [hunk 3]
```diff
@@ -103,6 +103,11 @@ At your console, the logs now contain the trace-id identifier:
...
+
+If you create a custom error that contains a _team_ property, this value will appear as _team_ tag on sentry. It is useful in cross applications, like jaiminho, us-emails, markito, etc, because we can group error by team
```
Comment (evandrocarrenho) [inline README.md:109]: "... we can group **errors** by team"

**Pull request merged**